package.json

```diff
@@ -1,89 +1,154 @@
-  "prettier": {
-    "printWidth": 100,
-    "singleQuote": true
-  },
-  "lint-staged": {
-    "*.js": [
-      "prettier-eslint --write",
-      "git add"
-    ]
-  },
   "keywords": [
     "react",
     "d3",
     "visualization",
     "vx",
     "charts"
   ],
+  "engines": {
+    "node": ">=10.15.0",
+    "yarn": ">=1.17.0"
+  },
   "author": "@hshoff",
-  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "build": "yarn run babel",
+    "build-one": "nimbus babel --clean",
+    "babel": "yarn run babel:cjs && yarn run babel:esm",
+    "babel:cjs": "nimbus babel --clean --workspaces=\"@vx/!(demo)\"",
+    "babel:esm": "nimbus babel --clean --workspaces=\"@vx/!(demo)\" --esm",
+    "clean": "rm -rf ./packages/*/*/{lib,esm}",
+    "format": "yarn run prettier --write",
+    "jest": "NODE_ENV=test nimbus jest --coverage --verbose",
+    "lint": "nimbus eslint",
+    "lint:fix": "yarn run lint --fix",
+    "prettier": "nimbus prettier",
+    "precommit": "lint-staged",
+    "setup": "yarn run build",
+    "test": "yarn run jest",
+    "docs": "yarn run docs:gen && node ./scripts/docs/index.js",
+    "docs:gen": "lerna run docs",
+    "prepare-release": "git checkout master && git pull --rebase origin master && yarn run docs && lerna updated",
+    "release": "yarn run prepare-release && lerna publish --exact"
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "yarn run format",
+      "git add"
+    ]
+  },
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-external-helpers": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-decorators": "^7.0.0",
-    "@babel/plugin-proposal-do-expressions": "^7.0.0",
-    "@babel/plugin-proposal-export-default-from": "^7.0.0",
-    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
-    "@babel/plugin-proposal-function-bind": "^7.0.0",
-    "@babel/plugin-proposal-function-sent": "^7.0.0",
-    "@babel/plugin-proposal-json-strings": "^7.0.0",
-    "@babel/plugin-proposal-logical-assignment-operators": "^7.0.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
-    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
-    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
-    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-syntax-import-meta": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^24.5.0",
-    "babel-plugin-istanbul": "^5.1.1",
+    "@airbnb/config-babel": "^2.0.0-alpha.2",
+    "@airbnb/config-eslint": "^2.0.0-alpha.2",
+    "@airbnb/config-jest": "^2.0.0-alpha.2",
+    "@airbnb/config-prettier": "^2.0.0-alpha.2",
+    "@airbnb/nimbus": "^2.0.0-alpha.2",
+    "@babel/core": "^7.5.0",
```

Comment:
The babel, eslint, prettier, and typescript deps are all installed by Nimbus. No need to install them here.